control/content/app.js

```diff
@@ -92,6 +92,12 @@ webviewPluginApp.controller("webviewPluginCtrl", ["$scope", "$log", "$timeout",
         $scope.urlValid = false;
         $scope.urlInValid = true;
       }
+
+      if(data.content.url.indexOf('?') != -1){
```

Comment:
shouldnt this be more like

`data.content.url += (data.content.url.indexOf('?')?'&':'?') +'view=inApp'`